src/Core/Framework/DataAbstractionLayer/DataAbstractionLayerException.php

```diff
@@ -88,6 +88,8 @@ class DataAbstractionLayerException extends HttpException
     public const UNSUPPORTED_QUERY_FILTER = 'FRAMEWORK__UNSUPPORTED_QUERY_FILTER';
     public const INVALID_SORT_DIRECTION = 'FRAMEWORK__INVALID_SORT_DIRECTION';
     public const PRODUCT_SEARCH_CONFIGURATION_NOT_FOUND = 'FRAMEWORK__PRODUCT_SEARCH_CONFIGURATION_NOT_FOUND';
+    public const DBAL_ENTITY_NOT_VERSIONABLE = 'FRAMEWORK__DBAL_ENTITY_NOT_VERSIONABLE';
```

Comment:
```suggestion
    public const ENTITY_NOT_VERSIONABLE = 'FRAMEWORK__DAL_ENTITY_NOT_VERSIONABLE';
```

Reply:
but DAL is also not mentioned in all the other constants. so it can also be removed